app/helpers/application_helper/button/storage_scan.rb

```diff
@@ -0,0 +1,7 @@
+class ApplicationHelper::Button::StorageScan < ApplicationHelper::Button::GenericFeatureButtonWithDisable
+  needs :@record
```

Comment:
@vecerek The button is converted correctly, but for some Datastores, when I try to "Perform SmartState Analysis", I'm getting error message:

![screenshot from 2017-03-06 16-32-04](https://cloud.githubusercontent.com/assets/1187051/23616618/a5d8ed8c-028a-11e7-943a-f7a23b45fc53.png)

I think we could avoid that, by also checking if the [Host is running and has valid credentials for Datastores](https://github.com/ManageIQ/manageiq/blob/ee0e90b66828c8d712918f584d5a944d8c575681/app/models/storage.rb#L367):

```ruby
Storage.find(@record.id).active_hosts_with_authentication_status_ok.empty?
```